code/parse/sexp.h

```diff
@@ -1142,6 +1183,59 @@ typedef struct sexp_variable {
 	char	variable_name[TOKEN_LENGTH];
 } sexp_variable;
 
+struct sexp_container
+{
+	// meta-character for containers in text replacement, etc.
+	static constexpr char DELIM = '&';
```

Comment:
I still think we should consider using a different metacharacter like `'^'`.

A FREDder might reasonably want to use `'&'` in player-facing text.

Reply:
BTW, in future containers PRs, I'm planning on replacing uses of the literal `'&'` with `sexp_container::DELIM`, which I added in this PR (it's not in PR #3398 ). Thoughts?

Reply:
I'm fine with that.

Reply:
Will the metacharacter's use here *prevent* its use in player-facing text?  As I recall, the $ can be used perfectly normally in most text; it only acquires special behavior when it is used as a color tag or placed before a variable.  If neither of these is true, the $ is displayed as expected.

If the & can be implemented the same way, then I think & is a better choice than ^.

Reply:
@Goober5000 Yes, if the text following a `&` is not a container name, then ,the text replacement code will skip over the `&`.

But why unnecessarily spend the computational overhead on checking whether the text following a `&` is a valid container name when we could just as easily use `^` as the meta-char instead?

Reply:
The computational overhead is going to be spent regardless.  Both & and ^ are valid "player-facing" characters.  Here's an example from BTA:

> Cluster weapon. Each mortar carries three explosive submunitions. A full salvo will blanket an area approximately 1 km^3 large.

Using & instead of ^ is simply my personal preference.  But regardless of the character chosen, we can't assume that all uses of that character are necessarily containers.

Reply:
Okay and yes, the text replacement code handles meta-char instances that are not followed by a container name. If there's some case where the new code doesn't handle it properly, we can discuss during the review for that PR, when it comes up.

We can stick with `&` as the meta-char.